src/redux/epics/event.epics.js

```diff
@@ -4,12 +4,21 @@ import { Observable } from 'rxjs';
 import 'rxjs/add/operator/mergeMap';
 
 const BASE_ENDPOINT = `https://www.eventbriteapi.com/v3/events/search/?token=${EVENTBRITE_API_KEY}&location.address=toronto&categories=`;
+const reCategoryPath = /^.*\/category\/\d\d\d$/;
```

Comment:
Are category ids always three digit numbers? It seems like they are, but you could replace it with this to be more flexible to match one or more numbers:
```
const reCategoryPath = /^\/category\/\d\d\d$/;
```
Is there a reason why you are matching `.*` at the beginning? The pathname is going to start with `/category` so you could leave this out.